genie-client/src/main/java/com/netflix/genie/client/apis/JobService.java

```diff
@@ -144,6 +144,24 @@
     @GET(JOBS_URL_SUFFIX + "/{id}/output/stderr")
     Call<ResponseBody> getJobStderr(@Path("id") String jobId);
 
+    /**
```

Comment:
The two methods above this one should now be removed, or is there some reason to keep them?

Reply:
They can be removed now. Fixed in following comming.